package.json

```diff
@@ -15,7 +15,7 @@
     "db:migrate": "postgrator",
     "db:down": "docker-compose down",
     "lint": "eslint --ext .ts,.js ./src",
-    "test": "cross-env TS_NODE_FILES=1 c8 tap --no-cov --ts"
+    "test": "cross-env TS_NODE_FILES=1 c8 tap -j2 --no-cov --ts"
```

Comment:
It seems that we are doing too much parallel tests, and the CI hasn't enough power to execute both all the tests and the postgres image in under 30 seconds